rs/consensus/src/consensus/purger.rs

```diff
@@ -287,6 +290,32 @@ impl Purger {
         );
     }
 
+    /// Equivocation proofs at and below the latest finalized height can
+    /// be purged from the pool.
+    fn purge_equivocation_proofs_by_finalized_height(
+        &self,
+        pool: &PoolReader,
+        finalized_height: Height,
+        changeset: &mut ChangeSet,
+    ) {
+        if pool
+            .pool()
+            .validated()
+            .equivocation_proof()
+            .height_range()
+            .is_some()
+        {
```

Comment:
Why is this needed? As I understand, when doing a `PurgeValidatedOfTypeBelow`, we anyway have an early return if the `Meta` table for the type doesn't exist (which is the same we are doing in this check here).

Reply:
It's not needed, but it does add unnecessary change actions into the set, which then show up in unrelated tests. Especially because equivocations are rare, I thought it might be better to only create change actions if they exist. 

But I'm open to getting rid of it as well, and merge the PR as it was before. Just my $0.02.

Reply:
I see, then I think I'm fine with it either way

Reply:
I see, then I'm fine with it either way